wrapanapi/google.py

```diff
@@ -226,15 +226,25 @@ def create_image(self, image_name, bucket_url, timeout=360):
         }
         operation = images.insert(project=self._project, body=data).execute()
         wait_for(lambda: self._nested_operation_wait(operation['name'], zone=False), delay=0.5,
-            num_sec=timeout, message=" Creating image {}".format(image_name))
+                 num_sec=timeout, message=" Creating image {}".format(image_name))
+
+    def delete_image(self, image_name, timeout=360):
+        """ Delete image
+        Args:
+            image_name: Unique name of image
+            timeout: time to wait for operation
```

Comment:
**OPTIONAL:** Provide the default value in comment.   The doc strings are used to document the API, and one could be looking at documentation without the source, so knowing the default would be good in that case.